Fix golint issues

`main.go`:

```
main.go:18:2: exported var Config should have comment or be unexported
main.go:35:6: exported type DownloadEvent should have comment or be unexported
main.go:41:6: exported type Series should have comment or be unexported
main.go:42:2: struct field Id should be ID
main.go:44:2: struct field TvdbId should be TvdbID
main.go:47:6: exported type Episode should have comment or be unexported
main.go:48:2: struct field Id should be ID
main.go:53:1: exported function Download should have comment or be unexported
```

`config/config.go`:

```
config/config.go:10:6: exported type Obj should have comment or be unexported
config/config.go:17:6: exported type HTTPConfig should have comment or be unexported
config/config.go:22:6: exported type MailgunConfig should have comment or be unexported
config/config.go:26:6: exported type Contacts should have comment or be unexported
config/config.go:28:1: exported function Read should have comment or be unexported
```

--- a/config/config.go
+++ b/config/config.go
@@ -7,6 +7,8 @@ import (
 	"github.com/BurntSushi/toml"
 )
 
+// Obj is a struct containing the different config
+// options for sonarrhook
 type Obj struct {
 	HTTP     HTTPConfig
 	Mailgun  MailgunConfig
@@ -14,17 +16,21 @@ type Obj struct {
 	Series   map[string]([]string)
 }
 
+// HTTPConfig contains the HTTP server config
 type HTTPConfig struct {
 	Host, Key string
 	Port      int
 }
 
+// MailgunConfig contains the Mailgun API and Email configs
 type MailgunConfig struct {
 	From, Domain, APIKey, PublicAPIKey string
 }
 
+// Contacts is a list of contacts and their emails
 type Contacts map[string]string
 
+// Read reads the config file and returns a config Obj
 func Read() Obj {
 	configfile := "./config/config.toml"
 	_, err := os.Stat(configfile)

--- a/main.go
+++ b/main.go
@@ -15,49 +15,53 @@ import (
 )
 
 var (
-	Config  config.Obj
-	Mailgun mailgun.Mailgun
+	c  config.Obj
+	mg mailgun.Mailgun
 )
 
 func main() {
 	log.Print("reading config file")
-	Config = config.Read()
+	c = config.Read()
 
-	http.HandleFunc("/download", Download)
+	http.HandleFunc("/download", download)
 
-	Mailgun = mailgun.NewMailgun(Config.Mailgun.Domain, Config.Mailgun.APIKey, Config.Mailgun.PublicAPIKey)
+	mg = mailgun.NewMailgun(c.Mailgun.Domain, c.Mailgun.APIKey, c.Mailgun.PublicAPIKey)
 
-	listenAddr := fmt.Sprintf("%s:%d", Config.HTTP.Host, Config.HTTP.Port)
-	log.Printf("listening on %s", listenAddr)
+	listenAddr := fmt.Sprintf("%s:%d", c.HTTP.Host, c.HTTP.Port)
+	log.Printf("listening on %s\n", listenAddr)
 	log.Fatal(http.ListenAndServe(listenAddr, nil))
 }
 
+// A DownloadEvent is an event sent by Sonarr that includes
+// info about the new episodes
 type DownloadEvent struct {
 	EventType string
 	Series    Series
 	Episodes  []Episode
 }
 
+// A Series is a sonarr series
 type Series struct {
-	Id          int
+	ID          int
 	Title, Path string
-	TvdbId      uint64
+	TvdbID      uint64
 }
 
+// An Episode is a file that was downloaded by Sonarr
 type Episode struct {
-	Id                                                           int
+	ID                                                           int
 	EpisodeNumber, SeasonNumber, QualityVersion                  int
 	Title, AirDate, AirDateUtc, Quality, ReleaseGroup, SceneName string
 }
 
-func Download(w http.ResponseWriter, r *http.Request) {
+func download(w http.ResponseWriter, r *http.Request) {
 	if r.Method != "POST" {
 		http.Error(w, http.StatusText(http.StatusBadRequest), http.StatusBadRequest)
 
 		return
 	}
 
-	if apiKey := r.URL.Query().Get("key"); apiKey != Config.HTTP.Key {
+	if apiKey := r.URL.Query().Get("key"); apiKey != c.HTTP.Key {
 		http.Error(w, http.StatusText(http.StatusForbidden), http.StatusForbidden)
 
 		return
@@ -91,26 +95,26 @@ func Download(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	log.Printf("got series [%d] %s", event.Series.Id, event.Series.Title)
+	log.Printf("got series [%d] %s", event.Series.ID, event.Series.Title)
 
 	var contactNames, recipients []string
 
-	contactNames, ok := Config.Series[strconv.Itoa(event.Series.Id)]
+	contactNames, ok := c.Series[strconv.Itoa(event.Series.ID)]
 	if !ok {
-		log.Printf("series id %d not in config", event.Series.Id)
+		log.Printf("series id %d not in config", event.Series.ID)
 		http.Error(w, http.StatusText(http.StatusBadRequest), http.StatusBadRequest)
 
 		return
 	}
 
 	for _, name := range contactNames {
-		if _, ok := Config.Contacts[name]; !ok {
+		if _, ok := c.Contacts[name]; !ok {
 			log.Printf("contact %s not in config", name)
 
 			continue
 		}
 
-		recipients = append(recipients, Config.Contacts[name])
+		recipients = append(recipients, c.Contacts[name])
 	}
 
 	if len(recipients) == 0 {
@@ -126,8 +130,8 @@ func Download(w http.ResponseWriter, r *http.Request) {
 		message.WriteString(fmt.Sprintf("+ S%dE%d - %s\n", ep.SeasonNumber, ep.EpisodeNumber, ep.Title))
 	}
 
-	transaction := mailgun.NewMessage(Config.Mailgun.From, title, message.String(), recipients...)
-	_, ID, err := Mailgun.Send(transaction)
+	transaction := mailgun.NewMessage(c.Mailgun.From, title, message.String(), recipients...)
+	_, ID, err := mg.Send(transaction)
 
 	log.Printf("message id %s", ID)
 	if err != nil {

